npm/private/utils.bzl

```diff
@@ -254,32 +254,18 @@ def _virtual_store_name(name, version):
 
 def _make_symlink(ctx, symlink_path, target_file):
     files = []
-    if ctx.attr.use_declare_symlink:
-        symlink = ctx.actions.declare_symlink(symlink_path)
-        ctx.actions.symlink(
-            output = symlink,
-            target_path = relative_file(target_file.path, symlink.path),
-        )
-        files.append(target_file)
-    else:
-        if _is_at_least_bazel_6() and target_file.is_directory:
-            # BREAKING CHANGE in Bazel 6 requires you to use declare_directory if your target_file
-            # in ctx.actions.symlink is a directory artifact
-            symlink = ctx.actions.declare_directory(symlink_path)
-        else:
-            symlink = ctx.actions.declare_file(symlink_path)
-        ctx.actions.symlink(
-            output = symlink,
-            target_file = target_file,
-        )
+    if not is_bazel_6_or_greater():
```

Comment:
Is the `make_symlink` util the best place to put this? 🤷

Reply:
Not the best place. I'll move it.

Reply:
Actually. I'll move this in a follow-up since the best place for the check doesn't exist yet in this commit.